### ISSUE TITLE
DAM - Digital Asset Management

### ISSUE THREAD

**Issue by jensr** [medin-marine/horizon-scanning#8]
Solutions for storing resources online in an accessible and consistent manner

Captured during MEDIN Open Meeting workshop on Horizon scanning

**Comment by OliverWilliamsDataManager** [comment]
Possible to also include internal only requirements for managing resources before publication?

**Comment by OliverWilliamsDataManager** [comment]
Big Picture folk I expect could contribute a lot

**Comment by GMDuncan** [comment]
Covers:

- Image detailed metadata standard (Standards working group) to ensure consistent metadata
- Service / API standard - see #14

**Comment by GMDuncan** [comment]
> Big Picture folk I expect could contribute a lot

Agreed - a UK imagery infrastructure (focusing on Benthic but no reason that it cant be broadened after the pilot!) is a key part of The Big Picture. Led by myself and Dan L

**Comment by jensr** [comment]
So I know people in MS have started work on this using Biigle for image analysis? Fomr what I understand, this platform allow oyu to "hook up" remote data - so perhaps a key question is whether DACs/MEDIN can enable direct image access for analysis?

**Comment by GMDuncan** [comment]
Would be interesting to have a stocktake of which DACs deal with still/digital imagery (avoiding geospatial remote sensing types such as backscatter as these are covered in their own standards!) to pull together relevant people, and look at commonalities and differences between imagery from different thematic domains.

**Comment by GMDuncan** [comment]
> So I know people in MS have started work on this using Biigle for image analysis? Fomr what I understand, this platform allow oyu to "hook up" remote data - so perhaps a key question is whether DACs/MEDIN can enable direct image access for analysis?

Yes indeed, and an easy way would be through something like #14 direct remote access of images.
